.github/workflows/pdf.yml

```diff
@@ -23,9 +23,9 @@ jobs:
       - name: Upload results
         uses: actions/upload-artifact@ea165f8d65b6e75b540449e92b4886f43607fa02 # v4
         with:
-          name: bluefin
+          name: contribute-cncf-io
           # The output filename can be specified with --output option
-          path: pdf/bluefin.pdf
+          path: pdf/contribute-cncf-io.pdf
```

Comment:
does this run as is? do we need to create an empty pdf folder, or can we just have this get created in the root?

Reply:
tbh, I don't know - i'm also fine with just getting rid of this as it was a carry-over for now

Reply:
@castrojo added it fairly recently, I didn't think it was a carry over, more a stubb/placeholder.

Reply:
Let's maybe disable it rather than remove it for now. Open an issue to revisit after next week.

Reply:
https://github.com/cncf/contribute-site/issues/44